src/main/java/flab/delideli/service/SessionLoginService.java

```diff
@@ -0,0 +1,20 @@
+package flab.delideli.service;
+
+import org.springframework.stereotype.Service;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+@Service
+public class SessionLoginService implements LoginService{
```

Comment:
이제 프로답게 이런 아주 사소한 것 하나하나 신경쓰는 장인정신이 필요합니다~

```suggestion
public class SessionLoginService implements LoginService {
```

Reply:
넵, 이런 사소한 부분을 생각하겠습니다.